pylint_odoo/test_repo/broken_module/models/broken_model.py

```diff
@@ -78,3 +78,33 @@ def my_method9(self):
     def my_method10(self):
         # A example of built-in raise without parameters
         raise ZeroDivisionError
+
+    def sql_method(self, ids, cr):
+        # This is the better way and should not be detected
+        self._cr.execute(
+            'SELECT name FROM account WHERE id IN %s', (tuple(ids),))
+        self.env.cr.execute(
+            'SELECT name FROM account WHERE id IN %s', (tuple(ids),))
+        cr.execute(
+            'SELECT name FROM account WHERE id IN %s', (tuple(ids),))
+
+    def sql_injection_method(self, cr, ids):
+        # SQL injection, bad way
+        self._cr.execute(
+            'SELECT name FROM account WHERE id IN %s' % (tuple(ids),))
+        self.env.cr.execute(
+            'SELECT name FROM account WHERE id IN %s' % (tuple(ids),))
+        cr.execute(
+            'SELECT name FROM account WHERE id IN %s' % (tuple(ids),))
+
+    def sql_injection_method3(self, ids, cr2):
+        # This cr.execute2 or cr2.execute should not be detected
+        self._cr.execute2(
+            'SELECT name FROM account WHERE id IN %s' % (tuple(ids),))
+        cr2.execute(
+            'SELECT name FROM account WHERE id IN %s' % (tuple(ids),))
+
+    def sql_injection_method5(self, ids):
+        var = 'SELECT name FROM account WHERE id IN %s'
+        values = ([1, 2, 3, ], )
+        self._cr.execute(var % values)  # sql injection too
```

Comment:
You could be the output of the tests [here](https://travis-ci.org/OCA/pylint-odoo/jobs/131135682#L594-L597)

```bash
pylint_odoo/test_repo/broken_module/models/broken_model.py:93: [E8103(sql-injection), TestModel.sql_injection_method] Use of "%" operator in execute database method. Better use parameters instead. - More info https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#no-sql-injection
pylint_odoo/test_repo/broken_module/models/broken_model.py:95: [E8103(sql-injection), TestModel.sql_injection_method] Use of "%" operator in execute database method. Better use parameters instead. - More info https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#no-sql-injection
pylint_odoo/test_repo/broken_module/models/broken_model.py:97: [E8103(sql-injection), TestModel.sql_injection_method] Use of "%" operator in execute database method. Better use parameters instead. - More info https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#no-sql-injection
pylint_odoo/test_repo/broken_module/models/broken_model.py:110: [E8103(sql-injection), TestModel.sql_injection_method5] Use of "%" operator in execute database method. Better use parameters instead. - More info https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#no-sql-injection
```

We have a expected number of errors by check [here](https://github.com/OCA/pylint-odoo/pull/29/files#diff-6177c602688f2a88312483059f6523d2R39)
```python
    'sql-injection': 4,
```

I hope to be clear